packages/mdc-fab/mdc-fab.scss

```diff
@@ -31,6 +34,27 @@
   @include mdc-fab--mini_;
 }
 
+.mdc-fab--extended {
+  @include mdc-typography(button);
+
+  display: inline-flex;
```

Comment:
Shouldn't these flex-related styles already be present on `.mdc-fab` (via the `base_` mixin)? They shouldn't need to be repeated here.

Reply:
Good point. Removed redundant styles from here.